server/db.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function() {
+
+  //Requring dependencies.
+  const dotenv = require('dotenv').load();
```

Comment:
Yes, it will error out, but my next question is how to use the 'process.env.DB_WHATEVER' without including the .env file. Brad said it would be with server config. So would we have to give the server the variables so that we would no longer have to use the .env file?